app/javascript/wikiapi.js

```diff
@@ -434,7 +437,8 @@ var wikidata = function(spec) {
   that.createPerson = function(personLabel, personDescription) {
     return that.ajaxAPI(true, 'wbeditentity', {
       new: 'item',
-      data: getPersonCreateData(personLabel, personDescription)
+      data: getPersonCreateData(personLabel, personDescription),
+      summary: wikidata.summary()
```

Comment:
I think this one should be `that.summary()`

Reply:
Ah good catch. Changed